2023/dotnetvale/data.json

```diff
@@ -0,0 +1,30 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "",
+      "",
+      ""
+    ]
+  },
+  "CommunityInfo": {
+    "Name": ".Net Vale",
+    "Country": "Brazil",
+    "Url": "https://www.sympla.com.br/evento/global-azure-bootcamp/1894910?_gl=1*yr5avl*_ga*OTk2NDE2MDUxLjE2NzYxNDc4NzQ.*_ga_KXH10SQTZF*MTY3NzQzMzUwMi4xMS4xLjE2Nzc0MzM3MTYuMC4wLjA",
+    "SessionizeApiId": "10800",
+    "Description_SessionizeApiId": "10800",
+    "SessionizeEventName": "global-azure",
+    "Logo": {
+      "Description": "Community logo is a .png file 500X500px named {foldername}.png.",
+      "File": "_template.png"
```

Comment:
You need to include your community logo! If you do not have one, some communities have made a simple image with their community name on it!